Packs/CommonPlaybooks/Playbooks/playbook-Account_Enrichment_-_Generic_v2.2.yml

```diff
@@ -0,0 +1,1103 @@
+- contextPath: ActiveDirectory.Users.userAccountControlFields.TRUSTED_FOR_DELEGATION
+  description: Whether the service account (the user or computer account) under which a service runs is trusted for Kerberos delegation.
+- contextPath: ActiveDirectory.Users.userAccountControlFields.SMARTCARD_REQUIRED
+  description: Whether to force the user to log in by using a smart card.
+- contextPath: ActiveDirectory.Users.userAccountControlFields.MNS_LOGON_ACCOUNT
+  description: Whether this is an MNS login account.
+- contextPath: ActiveDirectory.Users.userAccountControlFields.SERVER_TRUST_ACCOUNT
+  description: Whether this is a computer account for a domain controller that is a member of this domain. Works for *Windows Server 2012 R2*.
+- contextPath: IAM.Vendor
+  description: The returning results vendor.
+  type: unknown
+- contextPath: IAM.Vendor.action
+  description: The command name.
+- contextPath: IAM.UserProfile
+  description: The user profile.
+- contextPath: SailPointIdentityNow.Account
+  description: The IdentityNow account object.
+  type: unknown
+- contextPath: SailPointIdentityNow.Account.id
+  description: The IdentityNow internal id (uuid).
+- contextPath: SailPointIdentityNow.Account.name
+  description: Name of the identity on this account.
+- contextPath: SailPointIdentityNow.Account.identityId
+  description: The IdentityNow internal identity id.
+- contextPath: SailPointIdentityNow.Account.nativeIdentity
+  description: The IdentityNow internal native identity id.
+- contextPath: SailPointIdentityNow.Account.sourceId
+  description: Source id that maps this account.
+- contextPath: SailPointIdentityNow.Account.created
+  description: Timestamp when the account was created.
+- contextPath: SailPointIdentityNow.Account.modified
+  description: Timestamp when the account was last modified.
+- contextPath: SailPointIdentityNow.Account.attributes
+  description: Map of variable number of attributes unique to this account.
+- contextPath: SailPointIdentityNow.Account.authoritative
+  description: Indicates whether the account is the true source for this identity.
+- contextPath: SailPointIdentityNow.Account.disabled
+  description: Indicates whether the account is disabled.
+- contextPath: SailPointIdentityNow.Account.locked
+  description: Indicates whether the account is locked.
+- contextPath: SailPointIdentityNow.Account.systemAccount
+  description: Indicates whether the account is a system account.
+- contextPath: SailPointIdentityNow.Account.uncorrelated
+  description: Indicates whether the account is uncorrelated.
+- contextPath: SailPointIdentityNow.Account.manuallyCorrelated
+  description: Indicates whether the account was manually correlated.
+- contextPath: SailPointIdentityNow.Account.hasEntitlements
+  description: Indicates whether the account has entitlement.
+tests:
+- No tests (auto formatted)
```

Comment:
The test playbook for the original playbook "Account Enrichment - Generic v2.1 - Test"
We can add it here as well

Reply:
Correct. Adding it here .